### PR TITLE
Resolve Indentation Issue

### PR DESCRIPTION
**Pull request by NayanaNagaraj23**
Make changes in the following folders: os/arch/arm/src/armv7-a,os/arch/arm/src/common and os/arch/arm/include/armv7-a folders to resolve the indentation issue as per the rules

### CODE REVIEW

File: os/arch/arm/include/armv7-a/syscall.h [hunk 1]
```diff
@@ -195,13 +195,9 @@ static inline uintptr_t sys_call0(unsigned int nbr)
...
-	__asm__ __volatile__
-	(
-		"svc %1"
-		: "=r"(reg0)
-		: "i"(SYS_syscall), "r"(reg0)
-		: "memory", "r14"
-	);
+	__asm__ __volatile__("svc %1":"=r"(reg0)
+						 :"i"(SYS_syscall), "r"(reg0)
+						 :"memory", "r14");
```
Comment (sunghan-chang) [inline os/arch/arm/include/armv7-a/syscall.h:200]: Previous one looks better. Could you revert this?
Same as other `__asm__` code

File: os/arch/arm/src/armv7-a/arm_addrenv.c [hunk 2]
```diff
@@ -163,73 +163,70 @@
...
-  DEBUGASSERT(paddr);
+	paddr = (uintptr_t)(*virtptr) & PTE_SMALL_PADDR_MASK;
+	DEBUGASSERT(paddr);
 
 #ifdef CONFIG_ARCH_PGPOOL_MAPPING
-  /* Get the virtual address corresponding to the physical page address */
+	/* Get the virtual address corresponding to the physical page address */
 
-  virtptr = (uint32_t *)arm_pgvaddr(paddr);
+	virtptr = (uint32_t *) arm_pgvaddr(paddr);
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_addrenv.c:202]: Let's remove a space after typecasting. Please check others for the same.

**Comment by sunghan-chang** [comment]
1. Need to resolve conflicts
2. Need to modify typo resole -> resolve
3. Need to add commit description. It's our commit rule

**Comment by sunghan-chang** [comment]
And please find build breaks

**Comment by sunghan-chang** [comment]
and how about `fix coding rules violation for indentation`? `Issue` looks some problem...

**Comment by sunghan-chang** [comment]
@NayanaNagaraj23 @kishore-sn Is there any plan to resolve this PR?

**Comment by r-prabu** [comment]
@NayanaNagaraj23 avoid multiple commits for this. you need to amend the changes to the existing commit.

**Review by r-prabu — CHANGES_REQUESTED** [review]
Needs changes.

File: os/arch/arm/src/common/up_vfork.c [hunk 3]
```diff
@@ -149,7 +149,7 @@ pid_t up_vfork(const struct vfork_s *context)
 	child = task_vforksetup((start_t)(context->lr & ~1));
 	if (!child) {
 		sdbg("ERROR: task_vforksetup failed\n");
-		return (pid_t)ERROR;
+		return (pid_t) ERROR;
```
Comment (sunghan-chang) [inline os/arch/arm/src/common/up_vfork.c:152]: Need to revert this file all. Nothing is valid in this file.

File: os/arch/arm/src/common/up_usestack.c [hunk 4]
```diff
@@ -134,9 +134,9 @@ int up_use_stack(struct tcb_s *tcb, void *stack, size_t stack_size)
 	 */
 
 #ifdef CONFIG_ARCH_ARMV7A_FAMILY
-	top_of_stack = (uint32_t)tcb->stack_alloc_ptr + stack_size;
+	top_of_stack = (uint32_t) tcb->stack_alloc_ptr + stack_size;
```
Comment (sunghan-chang) [inline os/arch/arm/src/common/up_usestack.c:137]: same as previous. 
Please check others as well.

File: os/arch/arm/include/armv7-a/irq.h [hunk 5]
```diff
@@ -402,23 +406,18 @@ static inline irqstate_t up_irq_enable(void)
...
 {
-  __asm__ __volatile__
-    (
-      "msr    cpsr_c, %0"
-      :
-      : "r" (flags)
-      : "memory"
-    );
+	__asm__ __volatile__("msr    cpsr_c, %0"::"r"(flags)
+						 :"memory");
```
Comment (sunghan-chang) [inline os/arch/arm/include/armv7-a/irq.h:417]: Sorry but, this looks not good.
Even we must use a tab for indentation, we don't need to merge all separated lines.
Let's keep separate lines for each command
```
(tab)__asm__ __volatile__(
(tab)(tab)"msr  cpsr_c, %0"
(tab)(tab):
(tab)(tab): "r" (...)
....
..
```

Comment (sunghan-chang) [reply os/arch/arm/include/armv7-a/irq.h:417]: same as other __asm__ codes

File: os/arch/arm/src/common/up_vfork.c (hunk 3 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/common/up_vfork.c]: I will revert it

File: os/arch/arm/src/common/up_usestack.c (hunk 4 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/common/up_usestack.c]: I will check and correct it.

File: os/arch/arm/include/armv7-a/irq.h (hunk 5 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/include/armv7-a/irq.h]: Ok, I will change as other asm code.

File: os/arch/arm/include/armv7-a/irq.h [hunk 6]
```diff
@@ -381,15 +379,15 @@ static inline irqstate_t irqsave(void)
...
 
 /* Enable IRQs and return the previous IRQ state */
 
 static inline irqstate_t up_irq_enable(void)
 {
-  unsigned int cpsr;
-
+	unsigned int cpsr;
+	
   __asm__ __volatile__
```
Comment (sunghan-chang) [inline os/arch/arm/include/armv7-a/irq.h:391]: ASM code is not modified.... Please find my comment before I left.

Comment (NayanaNagaraj23) [reply os/arch/arm/include/armv7-a/irq.h:391]: Ok, I will modify it.

File: os/arch/arm/src/armv7-a/arm.h [hunk 7]
```diff
@@ -54,44 +54,44 @@
...
+#define PSR_A_BIT       (1 << 8)	/* Bit 8: Asynchronous abort mask */
+#define PSR_E_BIT         (1 << 9)	/* Bit 9:  Endianness execution state bit */
+#define PSR_GE_SHIFT      (16)	/* Bits 16-19: Greater than or Equal flags */
 #define PSR_GE_MASK       (15 << PSR_GE_SHIFT)
-                                    /* Bits 20-23: Reserved. RAZ/SBZP */
-#define PSR_J_BIT         (1 << 24) /* Bit 24: Jazelle state bit */
-#define PSR_IT01_SHIFT    (25)      /* Bits 25-26:  If-Then execution state bits IT[0:1] */
+/* Bits 20-23: Reserved. RAZ/SBZP */
+#define PSR_J_BIT         (1 << 24)	/* Bit 24: Jazelle state bit */
+#define PSR_IT01_SHIFT    (25)	/* Bits 25-26:  If-Then execution state bits IT[0:1] */
```
Comment (gSahitya-samsung) [inline os/arch/arm/src/armv7-a/arm.h:80]: Can you check the indentation here ?

File: os/arch/arm/src/armv7-a/arm_cpugating.c [hunk 8]
```diff
@@ -43,6 +43,35 @@
...
+
+/*-----------------------------------------------------------*/
+
+void ulPortInterruptUnLock(uint32_t key)
+{
+	__asm volatile("msr	cpsr_c, %0	\n"::"r"(key):"memory");
+}
+
+/****************************************************************************
+ * Name: arm_flash_handler
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_cpugating.c:74]: Why is there new code in this file? Its supposed to fix indentation for existing code only. Pls revert this file.

File: os/arch/arm/src/armv7-a/arm_cpuidlestack.c [hunk 9]
```diff
@@ -65,16 +65,14 @@
  ****************************************************************************/
 
 #if CONFIG_SMP_NCPUS > 1
-static const uint32_t *g_cpu_stackalloc[CONFIG_SMP_NCPUS] =
-{
-    0
-  , g_cpu1_idlestack
+static const uint32_t *g_cpu_stackalloc[CONFIG_SMP_NCPUS] = {
+	0, g_cpu1_idlestack
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_cpuidlestack.c:69]: Pls make this in 2 lines.

File: os/arch/arm/src/armv7-a/arm_cpupause.c [hunk 10]
```diff
@@ -171,7 +171,7 @@ int up_cpu_paused(int cpu)
    * requesting CPU.
    */
 
-  spin_unlock(&g_cpu_paused[cpu]);
+	spin_unlock(&g_cpu_paused[cpu]);
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_cpupause.c:174]: This file indentation has not been fixed. Pls check again.

File: os/arch/arm/src/armv7-a/arm_cpustart.c [hunk 11]
```diff
@@ -166,14 +163,14 @@ int arm_start_handler(int irq, void *context, void *arg)
...
 
 #ifdef CONFIG_SCHED_INSTRUMENTATION
-  /* Notify of the start event */
+	/* Notify of the start event */
 
-  sched_note_cpu_start(this_task(), cpu);
+	sched_note_cpu_start(this_task(), cpu);
 #endif
 
 #ifdef CONFIG_CPU_HOTPLUG
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_cpustart.c:176]: Fix the below line

File: os/arch/arm/src/armv7-a/arm_dataabort.c [hunk 12]
```diff
@@ -97,92 +97,88 @@ uint32_t system_exception_location;
...
+	return regs;				/* To keep the compiler happy */
 }
 
-#else /* CONFIG_PAGING */
+#else							/* CONFIG_PAGING */
 
 #include "section_config.h"
-SRAMDRAM_ONLY_TEXT_SECTION
-uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
+SRAMDRAM_ONLY_TEXT_SECTION uint32_t *arm_dataabort(uint32_t *regs, uint32_t dfar, uint32_t dfsr)
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_dataabort.c:181]: Fix all lines below

File: os/arch/arm/src/armv7-a/arm_addrenv.c [hunk 13]
```diff
@@ -262,92 +259,75 @@ static int up_addrenv_initdata(uintptr_t l2table)
...
-  if (ret < 0)
-    {
-      berr("ERROR: Failed to initialize .bss/.data region: %d\n", ret);
-      goto errout;
-    }
+	/* Initialize the shared data are at the beginning of the .bss/.data
+	 * region.
+	 */
+
+	ret = up_addrenv_initdata((uintptr_t) addrenv->data[0] & PMD_PTE_PADDR_MASK);
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_addrenv.c:303]: Let's remove a space after typecasting.

File: os/arch/arm/src/armv7-a/arm_addrenv.c [hunk 14]
```diff
@@ -543,124 +517,95 @@ ssize_t up_addrenv_heapsize(const group_addrenv_t *addrenv)
...
+	for (vaddr = CONFIG_ARCH_TEXT_VBASE, i = 0; i < ARCH_TEXT_NSECTS; vaddr += SECTION_SIZE, i++) {
+		/* Save the old L1 page table entry */
+
+		if (oldenv) {
+			oldenv->text[i] = mmu_l1_getentry(vaddr);
+		}
+
+		/* Set (or clear) the new page table entry */
+
+		paddr = (uintptr_t) addrenv->text[i];
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_addrenv.c:538]: same here

File: os/arch/arm/src/armv7-a/arm_addrenv.c [hunk 15]
```diff
@@ -543,124 +517,95 @@ ssize_t up_addrenv_heapsize(const group_addrenv_t *addrenv)
...
+	for (vaddr = CONFIG_ARCH_SHM_VBASE, i = 0; i < ARCH_SHM_NSECTS; vaddr += SECTION_SIZE, i++) {
+		/* Save the old L1 page table entry */
+
+		if (oldenv) {
+			oldenv->shm[i] = mmu_l1_getentry(vaddr);
+		}
+
+		/* Set (or clear) the new page table entry */
+
+		paddr = (uintptr_t) addrenv->shm[i];
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_addrenv.c:597]: Could you search `_t) `? Then you can find 29 lines.

File: os/arch/arm/src/armv7-a/arm_addrenv_shm.c [hunk 16]
```diff
@@ -82,136 +82,126 @@
...
 #ifdef CONFIG_ARCH_PGPOOL_MAPPING
-          /* Get the virtual address corresponding to the physical page
-           * address.
-           */
+			/* Get the virtual address corresponding to the physical page
+			 * address.
+			 */
 
-          l2table = (uint32_t *)arm_pgvaddr(paddr);
+			l2table = (uint32_t *) arm_pgvaddr(paddr);
```
Comment (sunghan-chang) [inline os/arch/arm/src/armv7-a/arm_addrenv_shm.c:141]: `*) `

**Review by sunghan-chang — COMMENTED** [review]
I left some comments. I think this PR is ok when we remove a space after typecasting. Maybe...
This is huge to review...

File: os/arch/arm/src/armv7-a/arm_undefinedinsn.c [hunk 17]
```diff
@@ -70,15 +70,15 @@ extern uint32_t system_exception_location;
 uint32_t *arm_undefinedinsn(uint32_t *regs)
 {
   _alert("Undefined instruction at 0x%x\n", regs[REG_PC]);
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/arm_undefinedinsn.c:72]: Line 72

File: os/arch/arm/src/armv7-a/cp15_cacheops.h [hunk 18]
```diff
@@ -156,30 +156,30 @@
 
 /* Set/way format */
 
-#define CACHE_WAY_SHIFT (3)     /* Bits 30-31: Way in set being accessed */
+#define CACHE_WAY_SHIFT (3)		/* Bits 30-31: Way in set being accessed */
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/cp15_cacheops.h:159]: check again. many mistakes till line  560

File: os/arch/arm/src/armv7-a/crt0.c [hunk 19]
```diff
@@ -88,18 +88,18 @@ static void sig_trampoline(void)
...
+      " mov  r0, r1\n"     /* R0=signo */
+      " mov  r1, r2\n"     /* R1=info */
+      " mov  r2, r3\n"     /* R2=ucontext */
+      " blx  ip\n"         /* Call the signal handler */
+      " pop  {r2}\n"       /* Recover LR in R2 */
+      " mov  lr, r2\n"     /* Restore LR */
+      " mov  r0, %0\n"     /* SYS_signal_handler_return */
+      " svc %1\n"          /* Return from the SYSCALL */
+      ::"i"(SYS_signal_handler_return),
+        "i"(SYS_syscall)
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/crt0.c:102]: Thesse are wrong. Pls use tabs.

File: os/arch/arm/src/armv7-a/mmu.h [hunk 20]
```diff
@@ -36,10 +36,10 @@
...
 
 /* References:
- *  "Cortex-A5� MPCore, Technical Reference Manual", Revision: r0p1,
- *   Copyright � 2010 ARM. All rights reserved. ARM DDI 0434B (ID101810)
- *  "ARM� Architecture Reference Manual, ARMv7-A and ARMv7-R edition",
- *   Copyright � 1996-1998, 2000, 2004-2012 ARM.
+ *  "Cortex-A5� MPCore, Technical Reference Manual", Revision: r0p1,
+ *   Copyright � 2010 ARM. All rights reserved. ARM DDI 0434B (ID101810)
+ *  "ARM� Architecture Reference Manual, ARMv7-A and ARMv7-R edition",
+ *   Copyright � 1996-1998, 2000, 2004-2012 ARM.
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/mmu.h:42]: mistakes from line 1006 to 1272

File: os/arch/arm/src/armv7-a/sctlr.h [hunk 21]
```diff
@@ -78,17 +78,17 @@
 
 /* Multiprocessor Affinity Register (MPIDR) */
 
-#define MPIDR_CPUID_SHIFT        (0)       /* Bits 0-1: CPU ID */
+#define MPIDR_CPUID_SHIFT        (0)	/* Bits 0-1: CPU ID */
```
Comment (kishore-sn) [inline os/arch/arm/src/armv7-a/sctlr.h:81]: check 440-463, 619, 771

File: os/arch/arm/src/common/go_os_start.c [hunk 22]
```diff
@@ -96,22 +96,18 @@ void go_os_start(void *pv, unsigned int nbytes)
...
+						 "\tbeq  2f\n"	/* (should not happen) */
+						 "\tbic  r0, r0, #3\n"	/* R0 = Aligned stackptr */
+						 "\tmovw r2, #0xbeef\n"	/* R2 = STACK_COLOR = 0xdeadbeef */
+						 "\tmovt r2, #0xdead\n" "1:\n"	/* Top of the loop */
+						 "\tsub  r1, r1, #1\n"	/* R1 nwords-- */
+						 "\tcmp  r1, #0\n"	/* Check (nwords == 0) */
+						 "\tstr  r2, [r0], #4\n"	/* Save stack color word, increment stackptr */
+						 "\tbne  1b\n"	/* Bottom of the loop */
+						 "2:\n" "\tmov  r14, #0\n"	/* LR = return address (none) */
+						 "\tb    os_start\n"	/* Branch to os_start */
```
Comment (kishore-sn) [inline os/arch/arm/src/common/go_os_start.c:110]: Too many tabs at start of lines.

File: os/arch/arm/src/armv7-a/arm_cpuidlestack.c (hunk 9 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_cpuidlestack.c]: Ok, I will check and change

File: os/arch/arm/src/armv7-a/arm_cpupause.c (hunk 10 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_cpupause.c:174]: I will fix the indentation.

File: os/arch/arm/src/armv7-a/arm_dataabort.c (hunk 12 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_dataabort.c:181]: I will recheck indentation and fix it.

File: os/arch/arm/src/armv7-a/arm_addrenv.c (hunk 13 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_addrenv.c]: Ok

File: os/arch/arm/src/armv7-a/arm_addrenv.c (hunk 14 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_addrenv.c]: I will check all the typecasting issue and change it

File: os/arch/arm/src/armv7-a/arm_addrenv.c (hunk 15 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_addrenv.c]: ok

File: os/arch/arm/src/armv7-a/arm_addrenv_shm.c (hunk 16 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_addrenv_shm.c:141]: ok I will check and change it

File: os/arch/arm/src/armv7-a/arm_gicv2.c [hunk 23]
```diff
@@ -468,24 +451,23 @@ void up_enable_irq(int irq)
...
-       * Clear-Enable Register (ICDISER)
-       */
+		/* Write '1' to the corresponding bit in the distributor Interrupt
+		 * Clear-Enable Register (ICDISER)
+		 */
 
-      regaddr = GIC_ICDICER(irq);
-      putreg32(GIC_ICDICER_INT(irq), regaddr);
+		regaddr = GIC_ICDICER(irq);
+		putreg32(GIC_ICDICER_INT(irq), regaddr);
```
[the remark this answers is not in the document]
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_gicv2.c:467]: ok , I will recheck and make necessary changes

File: os/arch/arm/src/armv7-a/arm_initialstate.c [hunk 24]
```diff
@@ -69,97 +69,91 @@
 
 void up_initial_state(struct tcb_s *tcb)
 {
-  struct xcptcontext *xcp = &tcb->xcp;
-  uint32_t cpsr;
+	struct xcptcontext *xcp = &tcb->xcp;
```
[the remark this answers is not in the document]
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_initialstate.c:72]: Ok

File: os/arch/arm/src/armv7-a/arm_l2cc_pl310.c [hunk 25]
```diff
@@ -68,31 +68,31 @@
 /* Number of ways depends on ARM configuration */
 
 #if defined(CONFIG_ARMV7A_ASSOCIATIVITY_8WAY)
-#  define PL310_NWAYS      8
-#  define PL310_WAY_MASK   0x000000ff
+#define PL310_NWAYS      8
+#define PL310_WAY_MASK   0x000000ff
```
[the remark this answers is not in the document]
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_l2cc_pl310.c:72]: ok, check and make required changes

File: os/arch/arm/src/armv7-a/arm_schedulesigaction.c [hunk 26]
```diff
@@ -55,7 +55,6 @@
 #include "irq/irq.h"
 #ifndef CONFIG_DISABLE_SIGNALS
 
-
```
[the remark this answers is not in the document]
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_schedulesigaction.c:58]: ok

File: os/arch/arm/src/armv7-a/arm_syscall.c [hunk 27]
```diff
@@ -61,7 +61,6 @@
 #define svcerr slldbg
 #define INDEX_ERROR (-1)
 
-
```
[the remark this answers is not in the document]
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/arm_syscall.c:64]: Ok check and make changes

File: os/arch/arm/src/common/go_os_start.c (hunk 22 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/common/go_os_start.c]: I will check and correct it

File: os/arch/arm/src/armv7-a/sctlr.h (hunk 21 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/sctlr.h:81]: Ok, will check and change

File: os/arch/arm/src/armv7-a/arm_schedulesigaction.c (hunk 26 above)
[the remark this answers is not in the document]
Comment (kishore-sn) [reply os/arch/arm/src/armv7-a/arm_schedulesigaction.c:58]: This is not yet fixed.

File: os/arch/arm/src/armv7-a/arm_syscall.c (hunk 27 above)
[the remark this answers is not in the document]
Comment (kishore-sn) [reply os/arch/arm/src/armv7-a/arm_syscall.c:64]: not fixed

File: os/arch/arm/src/armv7-a/arm_undefinedinsn.c (hunk 17 above)
Comment (kishore-sn) [reply os/arch/arm/src/armv7-a/arm_undefinedinsn.c:72]: not fixed

File: os/arch/arm/src/armv7-a/cp15_cacheops.h (hunk 18 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/cp15_cacheops.h:159]: I will check and update

File: os/arch/arm/src/armv7-a/crt0.c (hunk 19 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/src/armv7-a/crt0.c]: I will correct indentation

**Comment by kishore-sn** [comment]
I have fixed the missing issues in this PR. 
The PR looks good to merge now. 
@r-prabu Can you please approve it?

**Review by kishore-sn — APPROVED** [review]

File: os/arch/arm/include/armv7-a/irq.h [hunk 28]
```diff
@@ -199,10 +199,10 @@
 #define REG_SB              REG_R9
 #define REG_SL              REG_R10
 #ifdef CONFIG_ARM_THUMB
-  #define REG_FP            REG_R7
+#define REG_FP            REG_R7
 #else
-  #define REG_FP            REG_R11
-#endif /* CONFIG_ARM_THUMB */
+#define REG_FP            REG_R11
+#endif							/* CONFIG_ARM_THUMB */
```
Comment (r-prabu) [inline os/arch/arm/include/armv7-a/irq.h:205]: too many tabs. Is this intended for #endif? the same is present in many other places as well.

File: os/arch/arm/include/armv7-a/syscall.h [hunk 29]
```diff
@@ -85,23 +85,23 @@
 
 #ifndef CONFIG_SYS_RESERVED
 #error "CONFIG_SYS_RESERVED must be defined to have the value"
-#endif //CONFIG_SYS_RESERVED
+#endif							//CONFIG_SYS_RESERVED
```
Comment (r-prabu) [inline os/arch/arm/include/armv7-a/syscall.h:88]: some places it is used with /**/ and in some places // is used

File: os/arch/arm/include/armv7-a/irq.h (hunk 28 above)
Comment (NayanaNagaraj23) [reply os/arch/arm/include/armv7-a/irq.h:205]: The tabs are allocated as per the script.

File: os/arch/arm/include/armv7-a/syscall.h (hunk 29 above)
Comment (kishore-sn) [reply os/arch/arm/include/armv7-a/syscall.h:88]: I think this is ok. It was like this in the original code. We have only changed the indentation.

**Review by r-prabu — APPROVED** [review]

**Review by r-prabu — DISMISSED** [review]

**Review by r-prabu — APPROVED** [review]

**Pull request merged**